src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Presentation/ICompletionSetFactory.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.Presentation
+{
+    internal interface ICompletionSetFactory
```

Comment:
Can you add a doc comment comment what this is? It took me a bit of work to realize this was being called from the presenter.